site/pages/react/using-7702.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Using EIP-7702
+description: Learn how to use EIP-7702 to enable smart account features for EOAs
+---
+
+# Using EIP-7702
+
+## What is EIP-7702?
+
+EIP-7702 is an upcoming Ethereum upgrade set to launch in the coming months as part of the Pectra hardfork. Specifically, EIP-7702 enables Externally Owned Accounts (EOAs) to use smart contract account features. This upgrade marks a pivotal moment in Ethereum’s journey toward account abstraction, a breakthrough designed to make onchain experiences smoother, smarter, and more accessible. EIP-7702 doesn’t replace Ethereum’s existing AA standard, ERC-4337. EIP-7702 was designed to be complementary ERC-4337 and make it easier for those account abstraction features to reach users.
+
+- EIP-7702 enables EOAs to delegate control to smart contract accounts that can execute code directly from their addresses
+- It is compatible with the prevailing AA standard, ERC-4437, so existing wallets can become smart contract wallets without creating a new address and transferring assets
```

Comment:
It is compatible with the prevailing AA standard, ERC-4437, so **existing EOAs can enable smart account features** without creating a new address and transferring assets